kademlia/kademlia/network.py

```diff
@@ -7,14 +7,19 @@
 from twisted.internet.task import LoopingCall
 from twisted.internet import defer, reactor, task
 
-from kademlia.log import Logger
-from kademlia.protocol import KademliaProtocol
-from kademlia.utils import deferredDict, digest
-from kademlia.storage import ForgetfulStorage
-from kademlia.node import Node
-from kademlia.crawling import ValueSpiderCrawl
-from kademlia.crawling import NodeSpiderCrawl
+from log import Logger
+from protocol import KademliaProtocol
+from utils import deferredDict, digest
+from storage import ForgetfulStorage
+from node import Node
+from crawling import ValueSpiderCrawl
+from crawling import NodeSpiderCrawl
+from protocol import decodeTimestamp
+from protocol import encodeTimestamp
 
+import datetime
+
+from simplecrypt import encrypt, decrypt, DecryptionException
```

Comment:
Good point, I'll throw it in.
fyi: pip install simple-crypt

On Sun, Mar 29, 2015 at 10:12 AM, Mike Poirier <notifications@github.com>
wrote:

> In kademlia/kademlia/network.py
> <https://github.com/poiriermike/encryptedp2p/pull/22#discussion_r27356482>
> :
>
> >
> > +import datetime
> > +
> > +from simplecrypt import encrypt, decrypt, DecryptionException
>
> Are there extra libraries we have to download and install? If so, has the
> readme been updated to reflect this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/poiriermike/encryptedp2p/pull/22/files#r27356482>.
>